hil-test/tests/i2c.rs

```diff
@@ -71,7 +71,7 @@ mod tests {
             }
         }
 
-        assert_eq!(ctx.i2c.write(DUT_ADDRESS, &[]), Ok(()));
+        assert_eq!(ctx.i2c.write(DUT_ADDRESS, &[0xaa]), Ok(()));
```

Comment:
This is wrong, we need to support 0-byte writes

Reply:
ah - yeah now I see it was not always a no-op apparently